addons/event/models/event_event.py

```diff
@@ -705,6 +705,14 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
             for attendee in event.registration_ids.filtered(filter_func):
                 self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    def _get_date_begin_event_tz(self):
+        self.ensure_one()
+        return self.date_begin.astimezone(pytz.timezone(self.date_tz)).date()
```

Comment:
Does it work ? In my mind, `astimezone` will just set the timezone if there is none set before without changing the value.
Shouldn't we do: `pytz.utc.localize(self.date_begin).astimezone(pytz.timezone(self.date_tz)).date()`

Reply:
Is the stored date not always utc value?